README.md

```diff
@@ -72,5 +72,18 @@ the given file into your `.git/hook` directory:
 
     cp ./_devtools/git-hooks/post-commit .git/hooks/
 
+## Custom liquid tags
+
+Custom liquid tags are used to make linking to javadocs, properties, and documents easier
+
+| Tag   | Description            | Options                                                                         | Examples                                             | 
+| ----- | ---------------------- | ------------------------------------------------------------------------------- | ---------------------------------------------------- |
+| jlink | Creates Javadoc link   | Link text will be class name by default. Use `-f` for full package + class name | `jlink -f org.apache.accumulo.core.client.Connector` |
```

Comment:
Why not use `{% ... %}` in examples?

Reply:
Oops.  It would be better to include them. Fixed now